build_projects/shared-build-targets-utils/Publishing/AzurePublisher.cs

```diff
@@ -183,12 +183,37 @@ public string CalculateArchiveUploadUrl(string archiveFile, string channel, stri
             return $"{s_dotnetBlobRootUrl}{CalculateArchiveBlob(archiveFile, channel, version)}";
         }
 
-        public string CalculateArchiveBlob(string archiveFile, string channel, string version)
+        public static string CalculateArchiveBlob(string archiveFile, string channel, string version)
         {
             return $"{channel}/Binaries/{version}/{Path.GetFileName(archiveFile)}";
         }
 
-        public void DownloadFiles(string blobVirtualDirectory, string fileExtension, string downloadPath)
+        public static void DownloadFile(string blobFilePath, string localDownloadPath)
+        {
+            var blobUrl = $"{s_dotnetBlobRootUrl}{blobFilePath}";
+
+            using (var client = new HttpClient())
+            {
+                client.Timeout = TimeSpan.FromMinutes(1);
```

Comment:
Do we even need to change from the default timeout? Which is 100seconds (https://github.com/dotnet/corefx/blob/master/src/System.Net.Http/src/System/Net/Http/HttpClient.cs#L18). I wouldn't set it explicitly unless we run into an issue which we believe needs a custom timeout.